src/components/MARCTable/MARCTable.test.js

```diff
@@ -0,0 +1,352 @@
+import React from 'react';
+import {
+  render,
+  cleanup,
+  fireEvent,
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+import { Intl } from '../../../test/jest/__mock__/intl.mock';
+import { reduxFormMock } from '../../../test/jest/__mock__/reduxForm.mock';
+
+import { MARCTable } from './MARCTable';
+
+const onChange = jest.fn();
+const initialFields = [
+  {
+    order: 0,
+    field: { subfields: [{}] },
+  },
+];
+
+const renderMARCTable = fields => {
+  const component = () => (
+    <Intl>
+      <MARCTable
+        fields={fields}
+        onChange={onChange}
+      />
+    </Intl>
+  );
+
+  return render(reduxFormMock(component));
+};
+
+describe('MARC modifications table', () => {
+  afterEach(() => {
+    cleanup();
```

Comment:
Are you sure that cleanup is really needed? I was thinking that it should work by default.